app/build.gradle

```diff
@@ -105,6 +105,7 @@ dependencies {
 
     implementation "androidx.core:core-ktx:1.2.0-beta01"
     implementation "androidx.appcompat:appcompat:1.1.0"
+    implementation "androidx.fragment:fragment-ktx:1.1.0"
```

Comment:
added this dep so I could use the `commit` extension function in StatusListActivity